javascript/src/main/utils/aggregateUserMessageCount.js

```diff
@@ -0,0 +1,22 @@
+const aggregateUserMessageCount = (agg_messages, agg_slackUsers) => {
+    const userMessageCounts = [];
+
+    var i;
+    for(i = 0; i<agg_slackUsers.length;i++){
+        var count = 0;
+        var j;
```

Comment:
We should try to avoid using `var` in JavaScript and instead use `let` and `const` where appropriate.